bin/checkout

```diff
@@ -2,6 +2,9 @@
 set -ex
 cd "$(dirname "$0")/.."
 
+if [ ! -d lib ]; then
```

Comment:
Oops, it seems this isn't aligned with your style?
Maybe [ ! -d lib ] || mkdir lib ?